symbol_exporter/ast_symbol_extractor.py

```diff
@@ -67,6 +72,8 @@ def _symbol_stack_to_symbol_name(self):
         return ".".join(self.current_symbol_stack)
 
     def visit_Call(self, node: ast.Call) -> Any:
+        if hasattr(node.func, "id"):
```

Comment:
If call func is a name node then function is name space.
I see a bug here though - will also have to check if symbol has not been imported or else in the following example 'twos' will be added as an undeclared symbol.

```python
from abc import twos

b = twos(10)
```

Reply:
Fixed